src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const partner = person.sex === 'f'
+    ? 'husband'
+    : 'wife';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${person.name}`}</h2>
```

Comment:
Simplify it
```suggestion
      <h2 className="Person__name">My name is {name}</h2>
```

Reply:
I tried, but then linter writes "_require one JSX element per line_"